src/common/exif.cc

```diff
@@ -1065,7 +1065,10 @@ static bool _exif_decode_exif_data(dt_image_t *img, Exiv2::ExifData &exifData)
     if(FIND_EXIF_TAG("Exif.Photo.UserComment"))
     {
       std::string str = pos->print(&exifData);
-      dt_metadata_set_import(img->id, "Xmp.dc.description", str.c_str());
+      Exiv2::CommentValue value(str);
+      std::string str2(value.comment());
+      if(g_strcmp0("binary comment", str2.c_str()))
```

Comment:
This can be re-written as `if(str2 != "binary comment")`. I think it's more readable than `if(g_strcmp0())`, since with strcmp it's confusing that 0 (false) return value means string match.

Reply:
The way to learn a lot of stuff with 2 lines of code ! :smiley: 
Let's do it.